DigitalLearningSolutions.Web.Tests/Controllers/Login/LoginControllerTests.cs

```diff
@@ -342,5 +342,57 @@ public void Multiple_approved_accounts_at_same_centre_should_log_in()
             result.Should().BeRedirectToActionResult()
                 .WithControllerName("Home").WithActionName("Index");
         }
+
+        [Test]
+        public void
+            When_user_has_accounts_with_inactive_centres_only_check_for_centres_that_are_active()
```

Comment:
I think without context "check for centres" could mean anything in this title. I wonder if we could rename it such that it's more obvious what's happening.. Don't have any immediate good ideas though 😓

Reply:
Something like `When_user_has_accounts_with_inactive_centres_can_only_login_to_active_centres`?